platform/platform-filter-delegate/src/test/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilterTest.java

```diff
@@ -329,6 +327,11 @@ public void testRemoveSecurityFilter() throws IOException, ServletException {
     }
 
     @Test
+    public void testRemoveNullSecurityFilter() throws IOException, ServletException {
+        delegateServletFilter.removeSecurityFilter(null);
```

Comment:
In Spock there is! 🖖 
As far as I'm aware, the test will fail if an exception is thrown, but there's not an explicit way to specify this in JUnit.